exercises/go-counting/canonical-data.json

```diff
@@ -0,0 +1,170 @@
+{
+  "exercise": "go-counting",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Black corner territory on 5x5 board",
+      "property": "territory",
+      "input": {
+      "board": [
```

Comment:
I realise that I gave bad advice. Since this is a k/v pair under `input`, I would indent the `board` line and all the strings making up the board two spaces in